strings.txt

```diff
@@ -16858,3 +16858,955 @@
+    it = Gli utenti di MAPS.ME hanno apportato 50.000 modifiche alle mappe. Stiamo costruendo insieme le mappe più dettagliate al mondo. Unisciti a noi!
+    nb = MAPS.ME-brukere har foretatt 50 000 kartendringer. I fellesskap lager vi de mest detaljerte kartene i verden. Bli med du også!
+    zh-Hans = MAPS.ME 用户已经进行了 50 000 处地图修改。我们正在齐心协力打造世界上最详细的地图，加入我们吧！
+    th = ผู้ใช้ MAPS.ME ได้ทำการเปลี่ยนแปลงแผนที่เป็นจำนวนมากกว่า 50 000 ครั้งแล้ว เรากำลังทำให้แผนที่นี้มีรายละเอียดมากที่สุดในโลก มาร่วมมือกัน!
+    ja = MAPS.MEユーザーは50000マップの変更を行いました。我々は協力して、世界で最も詳細な地図を作っています。参加しませんか！
+    ro = Utilizatorii MAPS.ME au făcut 50.000 de modificări de hărți. Împreună alcătuim cele mai detaliate hărți din lume. Alăturați-vă comunității!
+    ar = قام مستخدمو MAPS.ME بإجراء 50000 تغيير على الخرائط. معًا سنحصل على الخرائط الأكثر تفصيلاً في العالم. انضم إلينا!
+    sk = Používatelia MAPS.ME urobili 50 000 zmien v mapách. Spoločne vytvárame najpodrobnejšie mapy na svete. Pridajte sa!
+    es = Los usuarios de MAPS.ME han realizado 50 000 cambios de mapas. Juntos estamos creando los mapas más detallados del mundo. ¡Únase a nosotros!
+    pl = Użytkownicy MAPS.ME dokonali 50 000 zmian w mapach. Wspólnie tworzymy najbardziej szczegółowe mapy na świecie. Przyłącz się do nas!
+    nl = MAPS.ME-gebruikers hebben 50.000 kaartwijzigingen doorgevoerd. Samen maken we de meest gedetailleerde kaarten ter wereld! Doe met ons mee!
+
+  [download_over_mobile_header]
+    comment = Заголовок ошибки при загрузке через сотовую связь
+    en = Download by using a cellular network connection?
+    ru = Загрузить через сотовую связь?
+    fr = Téléchargement avec une connexion réseau cellulaire ?
+    da = Downloader du ved hjælp af en mobilforbindelse?
+    id = Unduh dengan menggunakan koneksi jaringan seluler?
+    ko = 셀룰러 네트워크 접속을 사용하여 다운로드하시겠습니까?
+    sv = Ladda ned med mobildata?
+    tr = Hücresel bir ağ bağlantısı kullanarak indir?
+    uk = Завантажити за допомогою мобільної мережі?
+    vi = Tải xuống qua kết nối mạng di động?
+    hu = Letöltés mobilhálózati kapcsolat segítségével?
+    de = Download mit einer mobilen Netzwerkverbindung?
+    fi = Lataa käyttämällä puhelinverkkoyhteyttä?
+    cs = Stáhnout pomocí připojení přes mobilní síť?
+    it = Vuoi scaricare utilizzando una connessione di rete cellulare?
+    nb = Last ned via en mobil nettverkskobling?
+    zh-Hant = 用手機網絡連接下載嗎？
+    zh-Hans = 用手机网络连接下载吗？
+    th = ดาวน์โหลดโดยใช้การเชื่อมต่อเครือข่ายมือถือ?
+    ja = 携帯通信ネットワークで接続してダウンロードしますか？
+    ro = Descărcați utilizând o conexiune prin rețeaua de telefonie mobilă?
+    ar = تنزيل باستخدام اتصال الشبكة الخلوية؟
+    sk = Sťahovať prostredníctvom pripojenia na mobilnú sieť?
+    es = ¿Descargar con conexión de red móvil?
+    pl = Czy pobrać, używając połączenia z siecią komórkową?
+    nl = Downloaden via een mobiele gegevensverbinding?
+    pt = Baixar usando uma conexão de rede celular?
+
+  [download_over_mobile_message]
```

Comment:
+
> On Apr 26, 2016, at 4:04 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61080801>:
> 
> > +    cs = Stáhnout pomocí připojení přes mobilní síť?
> > +    it = Vuoi scaricare utilizzando una connessione di rete cellulare?
> > +    nb = Last ned via en mobil nettverkskobling?
> > +    zh-Hant = 用手機網絡連接下載嗎？
> > +    zh-Hans = 用手机网络连接下载吗？
> > +    th = ดาวน์โหลดโดยใช้การเชื่อมต่อเครือข่ายมือถือ?
> > +    ja = 携帯通信ネットワークで接続してダウンロードしますか？
> > +    ro = Descărcați utilizând o conexiune prin rețeaua de telefonie mobilă?
> > +    ar = تنزيل باستخدام اتصال الشبكة الخلوية؟
> > +    sk = Sťahovať prostredníctvom pripojenia na mobilnú sieť?
> > +    es = ¿Descargar con conexión de red móvil?
> > +    pl = Czy pobrać, używając połączenia z siecią komórkową?
> > +    nl = Downloaden via een mobiele gegevensverbinding?
> > +    pt = Baixar usando uma conexão de rede celular?
> > +
> > +  [download_over_mobile_message]
> Получается, нужно обязательно перепроверить все строки как минимум на точно такие же айдишники, а то полная ерунда получается.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61080801>